processing/src/main/java/io/druid/query/DefaultQueryMetrics.java

```diff
@@ -66,15 +65,6 @@ public void queryType(QueryType query)
   }
 
   @Override
-  public void interval(QueryType query)
```

Comment:
Interface change is possible, but status-quo emitting shouldn't be changed. You can change `dataSources()` method to `dataSourcesAndIntervals()` and change it's implementation body in `DefaultQueryMetrics` so that it emits what former `dataSource()` and `intervals()` methods did.